ui/src/app/pages/access/compliance-training-module-card-title.spec.tsx

```diff
@@ -87,16 +87,22 @@ const setup = (
   };
 };
 
+export const queryForRTTitle = () => {
+  return screen.queryByText(/responsible conduct of research training/i);
+};
+
+export const queryForCTTitle = () => {
+  return screen.queryByText(/controlled tier training/i);
+};
```

Comment:
@evrii I'm curious what your thoughts are on this. I extracted helpers in this test file to be used in both this test and in tests that rely on this component. Do you think there's a better way to reduce tests' reliance on component dependency internals in RTL tests?

Reply:
I think that this is fine, but what do you think about pulling these functions into a separate file called something like access-test-utils.js or even just test-utils since it would src/app/pages/access/test-utils.js. The later might be a good way of standardizing anytime we want to add common testing functions for pages.

Reply:
Good idea. I created an `access/test-utils.js` helper file.